pkg/providers/tinkerbell/stack/manifests/pbnj.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: apps/v1
```

Comment:
Might be worth considering treating these manifest similar to what we do for capi/capv/etc which are built/pushed to s3 in build tooling and then included in the bundle vs hardcoded into the cli.